Extension/src/LanguageServer/references.ts

```diff
@@ -159,7 +160,7 @@ export class ReferencesManager {
 
     initializeViews(): void {
         if (!this.viewsInitialized) {
-            this.referencesChannel = vscode.window.createOutputChannel(localize("c.cpp.references", "C/C++ References"));
+            this.referencesChannel = vscode.window.createOutputChannel(localize("c.cpp.peek.references", "C/C++ Peek References"));

```

Comment:
This should only create the output channel when a "peek references" command happens.  For the normal "find all references" command, this should not be created.